bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -49,7 +49,7 @@
   {% endif %}
 {% endblock %}
 
-{% macro card_block(class, tag_label, media_icon, title, ga_title, desc, image_url, include_highres_image, video_url, aspect_ratio, link_url, meta) -%}
+{% macro card_block(class, tag_label, media_icon, title, ga_title, desc, image_url, include_highres_image, video_url, video_id, aspect_ratio, link_url, meta) -%}
```

Comment:
Do we need both `video_url` and `video_id` still? Could we just have one parameter, maybe `youtube_id`?